tests/cli/test_pipeline_steps.py

```diff
@@ -14,7 +14,7 @@ class TestComponent:
         prefix: str = "example-prefix-"
 
     class TestPipeline:
-        components = [
+        components: ClassVar[TestComponent] = [
```

Comment:
might not hurt to do it here, but I would overall prefer to delay these changes until the Ruff adoption, so that we can examine the new diagnostics it provides

Reply:
Makes sense :)
Should I revert it?

I added it because it seemed inconsequential and generally regarded as good practice.


[PEP526](https://peps.python.org/pep-0526/#class-and-instance-variable-annotations) on `ClassVar`
[Pyright understands it](https://microsoft.github.io/pyright/#/type-concepts-advanced?id=pure-class-variables)

Reply:
aight, you have a point! :)